e2e-test/cromwell_azure_e2e_test.py

```diff
@@ -66,13 +66,15 @@ def get_workflow_metadata(app_url, workflow_id, bearer_token):
     return perform_get(workflow_endpoint, bearer_token)
 
 # workflow_ids is a deque of workflow ids
-def get_completed_workflow(app_url, workflow_ids, bearer_token, max_retries=4, sleep_timer=60 * 2):
+def get_completed_workflow(app_url, workflow_ids, bearer_token, max_retries=6, sleep_timer=60 * 2):
     success_statuses = ['Succeeded']
     throw_exception_statuses = ['Aborted', 'Failed']
+
+    current_retries = max_retries
     
     current_running_workflow_count = 0
     while workflow_ids:
-        if max_retries == 0:
+        if current_retries == 0:
             raise Exception(f"Workflow(s) did not finish running within retry window ({max_retries} retries)")
```

Comment:
`max_retries` in this log message was always 0 because we decremented `max_retries` during the run. I reassigned to `current_retries` and use that for tracking, so that we can log the real retry count here.